client-admin/public/index.html

```diff
@@ -23,7 +23,6 @@
   </head>
   <body class="viewport avenir">
     <div id="root"></div>
-    <script src="/dist/admin_bundle.js"></script>
```

Comment:
@patcon question: where is this replaced?

Reply:
Ah hey, that was a learning for me too: HtmlWebpackPlugin templates don't need it specified, as the [`inject` config](https://github.com/jantimon/html-webpack-plugin#options) determines where the script tag(s) are inserted. By default, it's at the bottom of the head tag

Reply:
I think we may actually want to make sure this comes after the `<div id="root></div>` container; Race conditions are possible otherwise.

Reply:
https://github.com/jantimon/html-webpack-plugin#options

I'm wondering if `inject` or `scriptLoading` config values would take care of this? (e.g. `inject: body` injects at _bottom_ of body, and `defer` in head miiiiight do something comparable or better? defer seems to be very commonly used in modern libraries, and the tactic of putting things at the bottom seems less common, but not sure what's changed in browsers to cause that)

While I don't know enough to gauge if the code does something that opens us up specifically to race conditions, I'd be a bit surprised if a plugin as widely-used as this didn't have _some_ way to mitigate this situation

Reply:
Yes, let's please use `inject: body`. I think you're right that `defer` may do the same thing, but for the sake of not having to worry about browser compatibility, let's just stick with the inserting in the body.